### ISSUE TITLE
Move away from 

### ISSUE THREAD

**Issue by pedro-stanaka** [prometheus/statsd_exporter#604]
## Summary

Can we move away from Circle CI and use only Github Actions? What was the history here and why we still use it?

## Why?

Seems like [Prometheus itself made the move 2y ago](https://github.com/prometheus/prometheus/pull/11235), but that is not a reason itself. I find depending on two CI systems quite clunky and we get much better integrated experience and faster CI if we use only one.

### Notes

@matthiasr if you can provide context here, would be nice. 

Relates to https://github.com/prometheus/statsd_exporter/issues/326

**Comment by SuperQ** [comment]
There was some plan to move away, and it got partly there. But work on migrating stopped due to lack of volunteer time.

IMO we should probably finish the migration, but I dislike the promci setup. The action mono-repo is clunky to use. I would prefer to refactor this so it's easier to work with.

**Comment by matthiasr** [comment]
Yeah, I never had the energy or enough pain to go through with it. IIRC the
influxdb exporter was one of the first migrated and may be a closer
analogue than prometheus/prometheus to serve as a template.

Tbh I've never really had a problem with the setup mainly because I don't
do much about it – I accept prombot's updates and that's about it. I'd
rather migrate to the common pattern now than wait for something better –
the more in line we are with the rest of the project the easier it will be
to follow or lead on any future developments.

**Comment by pedro-stanaka** [comment]
Closing this one. Thanks Ben!
